draft-ietf-taps-interface.md

```diff
@@ -1920,20 +1889,7 @@ Type:
 Applicability:
 : Connection, Message
 
-This property is a numeric (non-negative) value that represents an unbounded
-hierarchy of priorities. It can specify the priority of a Message, relative to
-other Messages sent over the same Connection or Connection Group (see
-{{groups}}), or the priority of a Connection, relative to other Connections in
-the same Connection Group.
-
-A Message with Niceness 0 will yield to a Message with Niceness 1, which will
-yield to a Message with Niceness 2, and so on. Niceness may be used as a
-sender-side scheduling construct only, or be used to specify priorities on the
-wire for Protocol Stacks supporting prioritization.
-
-This encoding of the priority has a convenient property that the priority
-increases as both Niceness and Lifetime decrease.
-
+See {{msg-niceness}}.
```

Comment:
I think here we still want the per-Connection Niceness as it was before it was merged with the per-Message Niceness.

I dug the old text out of the repo, and it said:
"This Property is a non-negative integer representing the relative inverse
priority of this Connection relative to other Connections in the same
Connection Group. It has no effect on Connections not part of a Connection
Group. As noted in {{groups}}, this property is not entangled when Connections
are cloned."

Reply:
ACK - and thanks for digging, this was very useful!